src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/CoreCaseDataService.java

```diff
@@ -877,4 +879,40 @@ public void saveCaseEvent(String authorisation, Long caseId, CaseEvent caseEvent
             );
         }
     }
+
+    public void savePaidInFull(Long caseId, PaidInFull paidInFull, String authorisation) {
+        try {
+            UserDetails userDetails = userService.getUserDetails(authorisation);
+
+            EventRequestData eventRequestData = eventRequest(SETTLED_PRE_JUDGMENT, userDetails.getId());
```

Comment:
not related to your PR but spelling should be: `SETTLED_PRE_JUDGEMENT`

Reply:
This is actually OK, Kerry confirmed, for us Judgement is without 'E'